### DISCUSSION TITLE
Puzzle: I have keys but no locks. I have space but no room. You can enter but can’t go outside. What am I?

### DISCUSSION THREAD

**Discussion by meviru** [meviru/badge-hunting#6] [Galaxy Trivia] [answered]
Puzzle: I have keys but no locks. I have space but no room. You can enter but can’t go outside. What am I?

**Comment by shynekomaid** [comment, marked as answer]
keyboard

**Reply by meviru** [reply]
Congratulations!! 👏👏 You've just earned the [Galaxy Brain](https://github.com/shynekomaid?tab=achievements&achievement=galaxy-brain) badge. Keep it up, and remember to give the repository a star to show your support! 🎉

**Reply by meviru** [reply]
If you like what I am doing please don't forget to give this repository a star. ⭐ It will motivate me to grow this community bigger and bigger.

**Reply by shynekomaid** [reply]
I just thought that the keyboard has locks, three of them - capslock scrolllock, numlock

**Reply by meviru** [reply]
@shynekomaid Great point! 
Indeed, keyboards have keys like Caps Lock, Scroll Lock, and Num Lock, which are referred to as locks even though they're not physical locks. Good thinking! We’re referring the physical locks here.

**Comment by PranjalTrivedi** [comment]
Mark Answer
